File: examples/rpg-car/android/src/main/java/com/fss/rpg/MainActivity.java
package com.fss.rpg;

import android.os.*;
import com.forcex.android.ForceXApp;

public class MainActivity extends ForceXApp
{
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        initialize(new RPGCarGame(),true);
    }
}

File: Natives/ForceX Core Android/app/src/main/java/com/fastsmartsystem/forcexcore/MainActivity.kt
package com.fastsmartsystem.forcexcore

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Example of a call to a native method
        val tv = findViewById<View>(R.id.sample_text) as TextView
        tv.setText("Hola mundo")

    }

    companion object {

        // Used to load the 'native-lib' library on application startup.
        init {
            System.loadLibrary("forcex")
        }
    }
}
